Library/Formula/mighttpd2.rb

```diff
@@ -16,14 +15,9 @@ class Mighttpd2 < Formula
 
   depends_on "ghc" => :build
   depends_on "cabal-install" => :build
-  depends_on "gmp"
```

Comment:
Are you sure? This runtime dependency seems to be for anything that depends on `ghc`.